presto-spi/src/main/java/com/facebook/presto/spi/predicate/Domain.java

```diff
@@ -81,6 +82,11 @@ public static Domain singleValue(Type type, Object value)
         return new Domain(ValueSet.of(type, value), false);
     }
 
+    public static Domain multipleValues(Type type, Object... values)
```

Comment:
This method is not used as varargs, so it'd be better to have it take a collection of some sort (List or Collection).

It's also problematic because the implementation assumes the argument list has at least 2 elements. API-wise, it's not clear what's the difference, say, between `singleValue(type, "x")` and `multipleValues(type, "x")`, or between `onlyNull(type)/notNull(type)` vs `multipleValues(type)`.